### PR TITLE
LPS-61680

### PR DESCRIPTION
**Pull request by hhuijser**

### CODE REVIEW

File: util-taglib/src/com/liferay/taglib/ui/AppViewEntryTag.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void setAssetTagClassPK(long assetTagClassPK) {
 	}
 
 	public void setAuthor(String author) {
-		_author = author;
+		_author = HtmlUtil.unescape(author);
```
Comment (samuelkong) [inline util-taglib/src/com/liferay/taglib/ui/AppViewEntryTag.java]: @daviddotzhang @daledotshan 

It doesn't make sense that we add an `unescape` here in the `.java` file and then turn around and do an escape in the `.jsp`.

Also, this change introduces a bug in `/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp`

**Comment by daledotshan** [comment]
Hey Sam,
I am just curious why other two unescape in the AppViewEntryTag.java is reasonable.

**Comment by samuelkong** [comment]
The other two unescape are also bad. There's several double unescape with title & description.

**Comment by samuelkong** [comment]
@daviddotzhang please check my changes. https://github.com/daviddotzhang/liferay-portal/pull/9

**Pull request closed without merging**